src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -44,18 +44,18 @@ internal sealed class Options
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Number of bits to hash into. Must be between 1 and 31, inclusive",
                 ShortName = "bits", SortOrder = 2)]
-            public int HashBits = HashingEstimator.Defaults.HashBits;
+            public int HashBits = HashingEstimator.Defaults.NumberOfHashBits;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Hashing seed")]
             public uint Seed = HashingEstimator.Defaults.Seed;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Whether the position of each term should be included in the hash",
                 ShortName = "ord")]
-            public bool Ordered = HashingEstimator.Defaults.Ordered;
+            public bool Ordered = HashingEstimator.Defaults.UseOrderedHashing;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Limit the number of keys used to generate the slot name to this many. 0 means no invert hashing, -1 means no limit.",
                 ShortName = "ih")]
-            public int InvertHash = HashingEstimator.Defaults.InvertHash;
+            public int InvertHash = HashingEstimator.Defaults.MaximumNumberOfInverts;
```

Comment:
`MaximumNumberOfSlotNames` maybe? I feel `MaximumNumberOfSlotNames` is too long but `Inverts` is not super clear. The word `inverts` is not explained in this doc string.

Reply:
This is not in the public surface, you should look at the extension/ColumnOptions for the actual xml description.

Reply:
Ah. Sure.